internal/route/repo/view.go

```diff
@@ -59,13 +65,26 @@ func renderDirectory(c *context.Context, treeLink string) {
 	}
 
 	var readmeFile *git.Blob
+	var readmeFileName string
 	for _, entry := range entries {
 		if entry.IsTree() || !markup.IsReadmeFile(entry.Name()) {
 			continue
 		}
 
 		// TODO(unknwon): collect all possible README files and show with priority.
 		readmeFile = entry.Blob()
+		readmeFileName = readmeFile.Name()
+		if ok, sourceEntry, sourcePath, err := isSymlink(entry, c); ok {
+			c.Data["IsSymlinkSourceExists"] = true
+			if err != nil {
+				c.Data["SourceTreePath"] = err.Error()
```

Comment:
Error should have its own variable, not mixed with regular variable.

Reply:
I want to achieve the following result: **when the source file is not present, the reason for the non-existence can be displayed**.

Of course, this is indeed a bad way, I will try to fix it.


<img width="502" alt="feature_05_the_source_file_not_exists" src="https://user-images.githubusercontent.com/7269913/80297944-ae877080-87ba-11ea-949c-af48cea142b7.png">